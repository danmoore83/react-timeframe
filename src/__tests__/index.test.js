import ReactTimeframe from '../index';

describe('<ReactTimeframe />', () => {

  it('renders correctly', () => {
    const component = shallow(<ReactTimeframe />);

    expect(component).toMatchSnapshot();
  });

})