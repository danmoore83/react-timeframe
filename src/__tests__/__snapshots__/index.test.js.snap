// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactTimeframe /> renders correctly 1`] = `ShallowWrapper {}`;
